lib/iris/tests/unit/fileformats/netcdf/test_Saver.py

```diff
@@ -458,6 +458,35 @@ def test_valid_range_and_valid_min_valid_max_provided(self):
                 saver.check_attribute_compliance(self.container, self.data)
 
 
+class Test__cf_coord_identity(tests.IrisTest):
+    def check(self, coord_system, units, expected_units):
+        coord = iris.coords.DimCoord([30, 45], 'latitude', units=units,
+                                     coord_system=coord_system)
+        saver = Saver('filename', 'NETCDF4')
```

Comment:
Argh! This is wrong. I was going to mock it out.

I had made _cf_coord_identity a static method in the hopes that I wouldn't have to instantiate the Saver class to test it, but I was getting an error
```
>>> coord = iris.coords.DimCoord([45], 'latitude', units='m')
>>> r = Saver._cf_coord_identity(coord)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: unbound method _cf_coord_identity() must be called with Saver instance as first argument (got DimCoord instance instead)
```